products/idn/api/patch-requests.md

```diff
@@ -0,0 +1,500 @@
+
+```text
+PATCH https://{tenant}.api.identitynow.com//v3/sources/:sourceId/schemas/:schemaId
+```
+
+```json
+[
+  {
+    "op": "add",
+    "path": "/attributes/-",
+    "value": {
+      "name": "location",
+      "type": "STRING",
+      "schema": null,
+      "description": "Employee location",
+      "isMulti": false,
+      "isEntitlement": false,
+      "isGroup": false
+    }
+  }
+]
+```
+
+This request uses the [PATCH Source Schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to add a new attribute, along with its details, to the end of the array of a source's schema's attributes. 
+
+This example uses the "-" after the path to indicate that the value will be added to the end of the array. You can specify the position in the array "/" following the path. For example, if there were a list of 3 attributes, you could specify "/attributes/2" and the request would add the new "location" attribute to the second position in the array. 
+
+## Specify a from 
+
+The "move" and "copy" operations allow you to remove or copy information from one path and add it to another path without your needing to specify the value, which could be an extensive array of information. To use the "move" and "copy" operations, you must specify a "from", a JSON Pointer representing the location you are moving or copying the value from. 
+
+This example request uses the [PATCH Source schema](https://developer.sailpoint.com/idn/api/v3/update-source-schema) endpoint to move an attribute, along with its details, from the beginning to the end of a source schema's array of attributes: 
+
+```text
+PATCH https://{tenant}.api.identitynow.com//v3/sources/:sourceId/schemas/:schemaId
+```
+
+```json
+
+ [
+  {
+    "op": "move", 
+    "from": "/attributes/0", 
+    "path": "/attributes/-"
+    }
+  ] 
+
+ ```
+
+ Instead of having to specify the value yourself, which could be an extensive array of information, you can use the "move" operation to move everything from one path to another. 
+
+## Apply the PATCH request header
+
+With an operation, a path, and a change to make, a PATCH request is almost ready. 
+
+To send the request as a PATCH, you must apply this special PATCH content type header: `Content-Type: application/json-patch+json`
+
+## Send your PATCH request 
+
+Once you have specified the operation you want to apply, the path you want to change, the change you want to make, and you have applied the PATCH content type header, you can send your PATCH request. 
+
+When the request is successful, the API will return the updated resource. 
+
+## Get started 
+
+Now you can use PATCH requests to make partially update resources. For more information about PATCH requests, refer to this [documentation](https://datatracker.ietf.org/doc/html/rfc6902#section-4.6).  For more information about the IdentityNow PATCH endpoints and which paths can be changed for each one, refer to their API specifications. 
```

Comment:
First sentence has an editing error in it.

Reply:
Yup, that should go to the top of the documentation - I'll fix that.